sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -96,4 +98,24 @@
 	public List<CasePersonDto> getDuplicates(@Valid CasePersonDto casePerson, @PathParam("reportDateThreshold") int reportDateThreshold) {
 		return FacadeProvider.getCaseFacade().getDuplicates(casePerson, reportDateThreshold);
 	}
+
+	///
+	@GET
+	@Path("/allPagingCases/{since}")
+	public Page<CaseDataDto> getPagingAllCases(@PathParam("since") long since, @QueryParam("page") int page, @QueryParam("size") int size ) {
+		return FacadeProvider.getCaseFacade().getPagingAllActiveCasesAfter(new Date(since), page, size);
+	}
+
+	@POST
+	@Path("/caseIndex")
+	public Page<CaseIndexDto> getIndexList(@RequestBody CaseCriteriaAndSorting caseCriteriaAndSorting, @QueryParam("page") int page, @QueryParam("size") int size ) {
```

Comment:
I'm not familiar with naming conventions for ReST endpoints, so bear with me if this doesn't make sense. Wouldn't it maybe make sense to give these new endpoints names that make their intentions very clear? Something like `/indexList` and `/detailedIndexList`?